src/TTextEdit.cpp

```diff
@@ -2385,9 +2384,8 @@ inline QString TTextEdit::convertWhitespaceToVisual(const QChar& first, const QC
             } else if ((value >= 0xFFF0 && value <= 0xFFF8) || value == 0xFFFE || value == 0xFFFF) {
                 //: Unicode codepoint in range U+FFFx - not a character.
                 return htmlCenter(tr("{noncharacter}")); break;
-            } else {
-                return htmlCenter(first);
             }
+            return htmlCenter(first);
```

Comment:
There is another `else-after-return` in the `default` for the `switch` in the `} else {` that starts two lines down from here.